src/Testing/InteractsWithMail.php

```diff
@@ -18,6 +20,40 @@ private function getMailer()
     {
         return $this->mailer ?: MailThief::getFacadeRoot();
     }
+    
+    /**
+     * @return MailThiefCollection
+     */
+    public function getMessages()
+    {
+        return $this->getMailer()->messages
+    }
+
+    /**
+     * @param array $emails
+     * @return MailThiefCollection
+     */
+    public function getMessagesFor(array $emails)
```

Comment:
Makes sense. I generally like using as strict of type hints as possible, which means not having arguments that could be different types. But, I see how this would be much nicer from a user perspective to take in both strings and arrays. Updating the code to match now.